doc/src/Howtorun.tex

```diff
@@ -102,24 +104,116 @@ \section{TEAL for RAVEN}
 </Economics>
 \end{lstlisting}
 
-As one can see, all the specifications of the \textbf{TEAL.CashFlow} module are given in the \xmlNode{Economics} block. The block accepts an attribute called \xmlAttr{verbosity},
+As one can see, all the specifications of the \textbf{TEAL.CashFlow} module are given in the \\\xmlNode{Economics} block. The block accepts an attribute called \xmlAttr{verbosity},
 which can range from 0 to 100, 0 meaning maximum debug verbosity and 100 meaning
 errors only. Setting the verbosity to 50 will output (in addition to errors) the
- NPV, IRR, PI or NPV\_mult. Inside the \xmlNode{Economics} block, there are two
+ NPV, IRR, PI, or NPV\_mult. Inside the \xmlNode{Economics} block, there are two
  types of blocks: \xmlNode{Global} and \xmlNode{Component}.
 
 An example of a cash flow is shown in Listing \ref{lst:CashFlowExample}. In the example, a cash flow called CAPEX is defined.
 In the example, the capital expenditure for a reference plant of capacity
 1'000'000'000 W is \$4 billion. The driver of this cash flow is the actual plant capacity in Watts. Building this plant has some economy
 of scale, so that a plant double the size does cost
-less than double the money ($X=0.64$). The example assums an overnight building cost of the reactor, i.e. $\alpha$ is zero for
-all years of the lifetime of the reactor except year zero, which is the construction year.
+less than double the money ($X=0.64$). The example assumes an overnight building cost of the reactor (i.e. $\alpha$ is zero for
+all years of the lifetime of the reactor except year zero, which is the construction year).
 
 \begin{lstlisting}[style=XML,morekeywords={anAttribute},caption=CashFlow input example., label=lst:CashFlowExample]
 <Capex name='CAPEX' tax='false' inflation='none'>
-    <driver>Plant_capacity</driver>
-    <alpha>-4000000000</alpha>
-    <reference>1000000000</reference>
-    <X>0.64</X>
+    <driver>Plant_capacity</driver> <!--The capacity of the design plant. This variable must be defined in RAVEN.-->
+    <alpha>-4000000000</alpha> <!--Reference overnight cost of the reference plant ($4 million)-->
+    <reference>1000000000</reference> <!--Reference capacity of the reference plant (1,000,000,000 W)-->
+    <X>0.64</X> <!--Scaling factor of 0.64 for the reference plant-->
 </Capex>
-\end{lstlisting}
\ No newline at end of file
+\end{lstlisting}
+
+Listing 6 gives an example of a RAVEN file corresponding to the input file example above. See the RAVEN user manual for more details.
```

Comment:
This example is somewhat suspect, as RAVEN can update with changed syntax and this example won't get automatically updated. I do like having the example; I guess in the future this could be a dynamically-linked example.

Reply:
I can also remove this and include it in the user guide eventually. I thought for now it would be nice to have until the user guide is complete.